components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/WorkflowManagementService.java

```diff
@@ -96,6 +95,10 @@ public interface WorkflowManagementService {
     void addWorkflow(Workflow workflowDTO,
                      List<Parameter> parameterList, int tenantId) throws WorkflowException;
 
+    String getTemplateId(String workflowId) throws  WorkflowException;
```

Comment:
Let's add java doc comments for the new methods. Check the above listed methods for reference.

Reply:
addressed the comment ref:https://github.com/wso2/carbon-identity-framework/pull/4519/commits/ce9bfa08202eb691d2a4f87aebd30fe3445f0fd5